src-c/dune

```diff
@@ -2,7 +2,5 @@
  (name checkseum_c)
  (public_name checkseum.c)
  (libraries bigarray optint checkseum.laolao)
- (wrapped false)
+ (implements checkseum)
  (flags (:standard -no-keep-locs)))
```

Comment:
You do not need to pass `no-keep-locs` manually in dune. Dune will do it automatically for you.